package.json

```diff
@@ -40,13 +40,13 @@
     "postinstall": "npx husky install"
   },
   "dependencies": {
-    "@astrojs/check": "^0.7.0",
-    "@astrojs/cloudflare": "^10.4.0",
-    "@astrojs/sitemap": "^3.0.5",
+    "@astrojs/check": "^0.9.3",
+    "@astrojs/cloudflare": "^12.0.0-beta.0",
```

Comment:
Interested in the changes here

Reply:
As far as I can tell ensures `astro:env` and correct `output` mode is supported (`hybrid` is now `static`).